### ISSUE TITLE
[Screen Reader - Windows Powershell]: A button must support one of these patterns: Invoke, Toggle or ExpandCollapse.

### ISSUE THREAD

**Issue by Chandresh1007** [microsoft/terminal#8483]
**User Experience:**
When standard controls from accessible technologies are not used and if the user interface elements are not used according to specification, Assistive Technologies (AT) will not be able to gather information about, activate(or set) and keep up to date on the status of user interface controls in the content.

**Test Environment:**
OS: Win 10 2004 Build (20262.1010)
App: Windows Terminal
App version: 1.4.3243.0
Tool: Accessibility Insights for Windows

**Repro Steps:**
1. Open Windows terminal app.
2. Press "Ctrl+Shift+T" to open multiple tabs.
3. Now run accessibility insight tool on the application.
4. Observe the issue.

**Actual Result:**
A button must support one of these patterns: Invoke, Toggle or ExpandCollapse.

**Expected Result:**
Modify the button to support exactly one of the following patterns:
 · Support the Invoke pattern if the button performs a command at the request of the user.
 · Support the Toggle pattern if the button can cycle through a series of up to three states.
 · Support the ExpandCollapse pattern if the button shows or hides additional content.

**MAS Reference:**
**MAS4.1.2**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B248054a6-5e68-4771-9e1e-242fb5025730%7D&cid=e71aec14-4173-423e-aecb-2472bbc751e8

![MAS4 1 2_button must support pattern_How to fix](https://user-images.githubusercontent.com/53220700/101023812-4b07c180-3599-11eb-89fa-ee433929599c.png)

![MAS4 1 2_button must support pattern_Details](https://user-images.githubusercontent.com/53220700/101023821-4cd18500-3599-11eb-9b14-76d7118fd980.png)

**Comment by zadjii-msft** [comment]
What the heck? I don't think there should be a button there...

I guess I don't even know where this button is. I'm pretty sure there isn't one in the XAML tree

**Comment by carlos-zamora** [comment]
The only buttons I'm seeing there are the Min/Max/Close buttons:
![Inspect.exe UIA Tree](https://user-images.githubusercontent.com/11050425/101069757-45858980-354f-11eb-9bf5-35e6773635cf.png)

Even using Accessibility Insights to traverse the tree, I'm seeing those same 3 buttons:
![Accessibility Insights UIA Tree](https://user-images.githubusercontent.com/11050425/101069957-8e3d4280-354f-11eb-8cd8-68ba26ecd3d1.png)

I just ran the automated tests from Accessibility Insights and I'm getting a different test failure:
![Accessibility Insights Test Failure](https://user-images.githubusercontent.com/11050425/101070229-ed02bc00-354f-11eb-85c8-11e8c0fa959f.png)

These might be different manifestations of the same issue? Not sure... Either way, settings `IsControlElement` to `true` shouldn't be too difficult, I think.

**Comment by DHowett** [comment]
@Chandresh1007 We cannot locate the button that you have reported here in Windows Terminal 1.5, 1.6, 1.7.

**Comment by Chandresh1007** [comment]
Issue is fixed. Verified on following Test Environment:
OS: Win 10 2004 Build (21301.1010)
App: Windows Terminal
App version: 1.5.10411.0
Tool: Accessibility Insights for Windows
Hence, closing the bug.

**Comment by Priyanka-Chauhan123** [comment]
Closing the bug as issue has been fixed.
